docs/tinlake/overview/interest.md

```diff
@@ -7,4 +7,85 @@ redirect_from:
   - /tinlake/
 ---
 
-# Interest
```

Comment:
General comment: I think, I'd have only the Examples in 
```
Code blocs
```
and have the formulas just in
`blue font`